src/main/docker/entrypoint.sh

```diff
@@ -15,7 +15,7 @@ if [ -n "${APPLICATION_EXTERNAL_CLASSPATH}" ]; then
     "
     mkdir /tmp/app
     unzip -qq app.war -d /tmp/app
-    cp -vR ${APPLICATION_EXTERNAL_CLASSPATH}/* /tmp/app/WEB-INF/lib
+    cp ${APPLICATION_DATASOURCE_EXTERNAL_DRIVER} /tmp/app/WEB-INF/lib
```

Comment:
maybe it's error?
we using folder for external dependency not only driver

Reply:
@ponaguynik can you check the reason for this change?

Reply:
I believe it's my mistake because of copypast from new version of files generated by JHipster and this change should be reverted.

Reply:
fixed